infoblox_client/connector.py

```diff
@@ -109,10 +114,21 @@ def _parse_options(self, options):
                 msg = "WAPI config error. Option %s is not defined" % attr
                 raise ib_ex.InfobloxConfigException(msg=msg)
 
-        for attr in ('host', 'username', 'password'):
-            if not getattr(self, attr):
-                msg = "WAPI config error. Option %s can not be blank" % attr
-                raise ib_ex.InfobloxConfigException(msg=msg)
+        def check(credentials):
+            for attr in credentials:
+                if not getattr(self, attr):
+                    msg = "WAPI config error. Option %s can not be blank" % attr
+                    raise ib_ex.InfobloxConfigException(msg=msg)
+
+        cerds_1 = {'host', 'username', 'password'}
+        cerds_2 = {'host', 'cert', 'key'}
```

Comment:
Those var names are not descriptive. Can we rename those sets to something like `creds_userpass_auth` and `creds_cert_auth`.

Reply:
I will modify the names as per your suggestion.